src/applications/static-pages/facilities/OtherFacilityListWidget.jsx

```diff
@@ -14,8 +14,13 @@ export default class OtherFacilityListWidget extends React.Component {
   }
 
   componentDidMount() {
-    this.request = apiRequest(`/facilities/va?ids=${this.props.facilities}`, {
-      apiVersion: 'v1',
+    this.request = apiRequest('/va', {
```

Comment:
Wondering: would we ever need to switch back to v1? If so we might consider adding a flipper for it in the future, should we need to go that route.

Reply:
We can't, they will discontinue v1 on June 1, 2024. Or they were planning to at least.

Reply:
But the other problem was this was v1 of _not_ our endpoint, but a VAOS endpoint that we shouldn't have been using since the start, but the names of the endpoints are confusing.

Reply:
Makes sense, thanks for clarifying.